frontend/public/components/mounted-vol.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import * as _ from 'lodash-es';
+
+import {
+  getVolumeType,
+  getVolumeLocation,
+  getVolumeMountPermissions,
+  getVolumeMountsByPermissions,
+} from '../module/k8s/pods';
+import {
+  VolumeIcon,
+  ResourceIcon,
+  EmptyBox,
+  SectionHeading,
+} from './utils';
+
+const ContainerLink = ({name, pod}) => <span className="co-resource-link co-resource-link--inline">
+  <ResourceIcon kind="Container" />
+  <Link to={`/k8s/ns/${pod.metadata.namespace}/pods/${pod.metadata.name}/containers/${name}`}>{name}</Link>
+</span>;
+
+const Volume = ({volumeMounts, pod}) => {
+  const kind = _.get(getVolumeType(volumeMounts.volume), 'id', '');
+  const loc = getVolumeLocation(volumeMounts.volume);
+  const name = volumeMounts.name;
+  const mountPermissions = getVolumeMountPermissions(volumeMounts);
+
+  return <div>
+    {volumeMounts.mounts.map((m, i) => <React.Fragment key={i}>
```

Comment:
If `name` is unique, it would be better to use that here. (I'm not sure offhand.)

```suggestion
    {volumeMounts.mounts.map((m) => <React.Fragment key={m.name}>
```